portal/src/main/scala/cromwell/pipeline/datastorage/dto/User.scala

```diff
@@ -12,5 +17,27 @@ final case class User(
   profilePicture: Option[ProfilePicture] = None
 )
 
+object User {
+  implicit lazy val userFormat: OFormat[User] = Json.format[User]
+}
+
 final case class UserId(value: String) extends MappedTo[String]
+
+object UserId {
+  implicit lazy val userIdFormat: OFormat[UserId] = Json.format[UserId]
```

Comment:
`UserId` should be formatted differently. You can find how this should be done at [this](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/16) PR.

Reply:
replaced with  ` implicit lazy val userIdFormat: Format[UserId] = implicitly[Format[String]].inmap(UserId.apply, _.value)` just as how in that PR